R/draw-dm.R

```diff
@@ -1,20 +1,20 @@
-#' Draw schema of a [`dm`]-object's data model
+#' Draw a schema of a [`dm`]-object's data model
 #'
 #' `cdm_draw()` draws a schema of the data model using `datamodelr` (which in turn uses `DiagrammeR`).
```

Comment:
```suggestion
#' `cdm_draw()` draws a schema of the data model using \pkg{DiagrammeR}.
```

Reply:
Removed reference to a data model. Is this correct?

```suggestion
#' `cdm_draw()` uses \pkg{DiagrammeR} to draw diagrams.
```

Reply:
This is much better than my suggestion.